src/site/stages/build/drupal/api.js

```diff
@@ -72,7 +73,28 @@ function getDrupalClient(buildOptions) {
       });
 
       if (response.ok) {
-        return response.json();
+        return response.text().then(data => {
```

Comment:
For context on this change: We wanted to log out the response body, but after running `response.body()`, we couldn't run `response.text()` because the body was already "consumed." Turns out it's a thing.

The idea here was to just take the `.text()`, try to parse it, and then log the error and context if it fails.